.gitignore

```diff
@@ -9,3 +9,6 @@ _build/
 .*.swp
 *.swo
 stdout
+*.iml
+.idea
```

Comment:
I think this kind of stuff should be left out of this project's `.gitignore`, and that it belongs to a user's individual `$HOME/.gitignore` file. Otherwise we'd have to explicitly ignore temporary files from every editor, in every project.